### ISSUE TITLE
Local Modpack Import (CurseForge .zip, Modrinth .mrpack)

### ISSUE THREAD

**Issue by GaucovyAero** [PojavLauncherTeam/PojavLauncher#5555]
### Describe the request.

Add the ability to import local modpack files, maybe a button on top of "Search for modpacks" saying "Import local modpack" or "Import local file".

**Comment by Marshall2439** [comment]
Yeah it SHOULD be a feature especially after android 13 because not most of people have access to secret files app or adb

**Comment by GaucovyAero** [comment]
> Yeah it SHOULD be a feature especially after android 13 because not most of people have access to secret files app or adb

this is possible by using com.marc.files and clicking on the three lines - pojavlauncher

**Comment by Marshall2439** [comment]
> > Yeah it SHOULD be a feature especially after android 13 because not most of people have access to secret files app or adb
> 
> this is possible by using com.marc.files and clicking on the three lines - pojavlauncher

I know bro since android 13 I'm using that app only problem is it is not user friendly interface (atleast for me) i just prefer using wireless adb and Zarchiever

**Comment by SolDev69** [comment]
Was planning to experiment with adding this in SCL, still have to look into that.
